interface/patient_file/report/custom_report.php

```diff
@@ -804,7 +803,9 @@ function next_prev(action){
                 if($couch_docid && $couch_revid){
                   $url_file = $d->get_couch_url($pid,$encounter);
                 }
-                // Collect filename and path
+                //Extract the extension by the mime/type and not the file name extension
+                $image_data = getimagesize($url_file);
+                $extension = image_type_to_extension($image_data[2]);               // Collect filename and path
```

Comment:
Not reliable to use $url_file to find a file. Instead recommend doing this with $from_file after line 827.
``````
// Accurately collect the proper extension
$image_data = getimagesize($from_file);
$extension = image_type_to_extension($image_data[2]);
``````